onsp_co2/models/carbon_factor.py

```diff
@@ -169,15 +254,13 @@ def get_value_infos_at_date(self, date=None) -> dict:
     # --------------------------------------------
 
     def action_see_child_ids(self):
-        self.ensure_one()
-        return {
-            "name": _("Child factors for %s", self.display_name),
-            "type": "ir.actions.act_window",
-            "res_model": "carbon.factor",
-            "views": [(False, "tree"), (False, "form")],
-            "domain": [("parent_id", "=", self.id)],
-            "target": "current",
-            "context": {
-                **self.env.context,
-            },
-        }
+        return self._generate_action(title="Child factors", model="carbon.factor")
+        
+    def action_see_chart_of_account_ids(self):
+        return self._generate_action(title="Chart of Account", model="account.account")
+
+    def action_see_product_ids(self):
+        return self._generate_action(title="Product", model="product.template")
+        
+    def action_see_product_categ_ids(self):
+        return self._generate_action(title="Product Category", model="product.category")
```

Comment:
Could you add a translation wrapper (`_`) on titles please ?

Reply:
fix in https://github.com/MyCityCO2/mycityco2-engine/pull/13/commits/3d7e68249ab3bf6ec3e3740a7ece184995375887